src/main/java/com/example/KitchenProject/QuantityUnit.java

```diff
@@ -0,0 +1,11 @@
+package com.example.KitchenProject;
+
+public enum QuantityUnit {
+    GRAM,
+    SPOON,
```

Comment:
```suggestion
    TEA_SPOON,
```
just to be more precise;)